src/Mono.Android/Android.Runtime/JNIEnvInit.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+using Java.Interop;
+using Java.Interop.Tools.TypeNameMappings;
+
+namespace Android.Runtime
+{
+	static internal class JNIEnvInit
```

Comment:
Can this be moved to the other new assembly?  (I think I asked this before, but forgot the answer.)

Reply:
It is included in the new assembly for .NET, for classic it remains in Mono.Android, thus the source is here and the new assembly includes it with a relative path.